wp-cache.php

```diff
@@ -3688,16 +3686,29 @@ function wpsc_enable_preload() {
 	wp_schedule_single_event( time() + 10, 'wp_cache_full_preload_hook' );
 }
 
+function wpsc_get_post_types() {
+
+	$preload_type_args = apply_filters( 'wpsc_preload_type_args', array(
+		'public'             => true,
+		'publicly_queryable' => true
+	) );
+
+	$post_types = (array) apply_filters( 'wpsc_preload_post_types', get_post_types( $preload_type_args, 'names', 'or' ));
+
+	return join( "', '", array_map( 'esc_sql', $post_types ) );
```

Comment:
It makes sense to this function returns string with all quotes . I mean:
`'post', 'page',...` instead `post', 'page`. So, I propose:
`return "'" . join( "', '", array_map( 'esc_sql', $post_types ) ) . "'";`